exporter/keystoneexporter/keystone.go

```diff
@@ -231,8 +231,8 @@ func GetKsGatewayUrl() (string, error) {
 
 	streamName := fmt.Sprintf(streamNameFmt, env)
 
-	// "https://ksgateway-${REGION}.prod.netflix.net/REST/v1/stream/${STREAM_NAME}"
-	return fmt.Sprintf("http://ksgateway-%s.prod.netflix.net/REST/v1/stream/%s", region, streamName), nil
+	// "https://ksgateway-titus-metrics-${REGION}.prod.netflix.net/REST/v1/stream/${STREAM_NAME}"
+	return fmt.Sprintf("http://ksgateway-titus-metrics-%s.prod.netflix.net/REST/v1/stream/%s", region, streamName), nil
```

Comment:
would it be better to make the whole property an env var, and then have the env var constructed in systemd or similar?

Reply:
otel actually has a supported way for configuring exporters (and other plugins).  I'll have a look and see if it can reference an envvar or something so we can change this more easily.